api/openapi.gen.yaml

```diff
@@ -63,6 +69,27 @@ components:
         org_id:
           type: string
       type: object
+    v1.BadRequestError:
+      properties:
+        msg:
+          type: string
+        request_id:
+          type: string
+      type: object
+    v1.InternalRequestError:
+      properties:
+        msg:
+          type: string
+        request_id:
+          type: string
+      type: object
+    v1.NotFoundError:
+      properties:
+        msg:
+          type: string
+        request_id:
+          type: string
+      type: object
```

Comment:
I don't see the use of the interface in the schema, I would expect that there will be some kind of common component that matches the interface, but here it uses structs directly. Can you explain in more detail what was the motivation?

Because RequestId is not even available through the interface, yet it is mapped here directly from the type.